pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stock-indicators"
+authors = [{name = "Dave Skender"}]
+maintainers = [{name="Dong-Geon Lee"}]
+description = "Stock Indicators for Python.  Send in historical price quotes and get back desired technical indicators such as Stochastic RSI, Average True Range, Parabolic SAR, etc.  Nothing more."
+readme = "README.md"
+license = {text = "Apache 2.0"}
+classifiers = [
+    'Intended Audience :: Developers',
+    'Topic :: Office/Business :: Financial',
+    'Topic :: Office/Business :: Financial :: Investment',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'License :: OSI Approved :: Apache Software License',
+    'Operating System :: OS Independent',
+    'Operating System :: POSIX',
+    'Operating System :: Unix',
+    'Operating System :: MacOS',
+    'Typing :: Typed'
+]
+requires-python = ">=3.8"
+dependencies = [
+    "pythonnet>=3.0.0",
+    "typing_extensions>=4.0.0"
+]
+keywords = ["egg", "bacon", "sausage", "tomatoes", "Lobster Thermidor"]
```

Comment:
@DaveSkender Can you recommend some [keywords](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#keywords) for better SEO(?) at PyPi?

Reply:
Here's the tags from the .NET project for some ideas.  I put in too many.  Any of the tags here that are _not indicator names_ for framework specific should work.  I front-loaded the more relevant ones.

https://github.com/DaveSkender/Stock.Indicators/blob/6c70e30eff7e9b8bbd0e37a3fbf006e585346b34/src/Indicators.csproj#L24-L35